integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/ElasticsearchExtensionIT.java

```diff
@@ -171,6 +172,21 @@ public void query() {
 				.isInstanceOf( SearchException.class );
 	}
 
+	@Test
+	public void tookAndTimedOut() {
+		StubMappingScope scope = indexManager.createScope();
+
+		SearchQuery<DocumentReference> genericQuery = scope.query()
+				.predicate( f -> f.matchAll() )
+				.toQuery();
+
+		ElasticsearchSearchQuery<DocumentReference> esQuery = genericQuery.extension( ElasticsearchExtension.get() );
+		ElasticsearchSearchResult<DocumentReference> result = esQuery.fetchAll();
+
+		assertNotNull( result.getTook() );
+		assertNotNull( result.getTimedOut() );
```

Comment:
You could even assert it's false, in this case :)

Reply:
Yes, I thought about that, but I was not sure about to do that.